spec/components/candidate_interface/application_dashboard_course_choices_component_spec.rb

```diff
@@ -275,6 +275,32 @@
     end
   end
 
+  context "when there's an offer with a standard SKE condition" do
+    let(:application_choice) { create(:application_choice, :offered, offer:, course_option:) }
+    let(:application_form) { application_choice.application_form }
+    let(:offer) { build(:offer, :with_ske_conditions) }
+    let(:course_option) { build(:course_option, course:) }
+    let(:course) { build(:course, course_subjects:) }
+    let(:course_subjects) { [build(:course_subject, subject:)] }
+    let(:subject) { build(:subject, :non_language) } # rubocop:disable RSpec/SubjectDeclaration
```

Comment:
given our ongoing involvement in the world of education, perhaps this is one to add to the `rubocop.yml`

Reply:
I'm not sure, that's one for the team.  I think it's a good warning in general, I just couldn't think of another word to use.